packages/react-composites/src/composites/CallComposite/components/buttons/People.tsx

```diff
@@ -50,8 +43,6 @@ export const People = (props: ControlBarButtonProps): JSX.Element => {
       onRenderOffIcon={onRenderOffIcon ?? icon}
       onClick={onClick}
       styles={styles}
-      /* @conditional-compile-remove(rooms) */
-      disabled={disabled}
```

Comment:
Do we have a chance to test this, can consumer show the partcipant list without problem?

Reply:
Consumer can see the participants now